javascript-cont/destructuring.md

```diff
@@ -102,9 +118,20 @@ And we can use the syntax to create a new array of all the elements from an arra
 ```javascript
 const dogs = ['Fido', 'Buddy', 'Snoopy', 'Sparky'];
 
-const [ , ...otherDogs ] = dogs;
+const [, ...otherDogs ] = dogs;
```

Comment:
Did you want the white space after `...otherDogs` ?
```suggestion
const [, ...otherDogs] = dogs;
```

Reply:
I did not, thanks for catching the mismatch!